src/ng-generate/table-prompter/index.ts

```diff
@@ -752,5 +758,6 @@ function getUserConfigQuestions(allAnswers: any, tree: Tree, options: Schema): Q
         requestAspectModelVersionSupport,
         requestCustomStyleImports,
         requestOverwriteFiles,
+        requestOptionalMaterialTheme,
```

Comment:
Not at last maybe before the "requestCustomStyleImports". Hmm or something before?

Reply:
done